src/main/java/graphql/language/AbstractNode.java

```diff
@@ -29,4 +31,31 @@ public void setComments(List<Comment> comments) {
         assertNotNull(comments, "You must provide non null comments");
         this.comments = comments;
     }
+
+    protected boolean isEqualTo(String thisStr, String thatStr) {
```

Comment:
this is confusing: this is just a string equals right? if we want it as a helper method it should be moved to some other place imho

Reply:
sure - just that it was in 30 places before - all derived from this class.  So I made the orther places simplier via inheritance.  We can use static compostion if you like

Reply:
fixed - moved to NodeUtil